<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Automations on TestNg Basics" duration-ms="47" started-at="2018-10-11T06:44:10Z" finished-at="2018-10-11T06:44:10Z">
    <groups>
    </groups>
    <test name="Different testng features test" duration-ms="47" started-at="2018-10-11T06:44:10Z" finished-at="2018-10-11T06:44:10Z">
      <class name="com.test.ParameterAnnotationTest">
        <test-method status="PASS" signature="yahooLoginTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.test.ParameterAnnotationTest@1e127982]" name="yahooLoginTest" duration-ms="0" started-at="2018-10-11T06:44:10Z" finished-at="2018-10-11T06:44:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://login.yahoo.com/]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[naveen@yahoo.com]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- yahooLoginTest -->
      </class> <!-- com.test.ParameterAnnotationTest -->
      <class name="com.test.TestNgFeatures">
        <test-method status="PASS" signature="LoginTest()[pri:0, instance:com.test.TestNgFeatures@10bdf5e5]" name="LoginTest" duration-ms="0" started-at="2018-10-11T06:44:10Z" finished-at="2018-10-11T06:44:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="SearchTest()[pri:0, instance:com.test.TestNgFeatures@10bdf5e5]" name="SearchTest" duration-ms="0" started-at="2018-10-11T06:44:10Z" finished-at="2018-10-11T06:44:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SearchTest -->
        <test-method status="PASS" signature="HomepageTest()[pri:0, instance:com.test.TestNgFeatures@10bdf5e5]" name="HomepageTest" duration-ms="0" started-at="2018-10-11T06:44:10Z" depends-on-methods="com.test.TestNgFeatures.LoginTest" finished-at="2018-10-11T06:44:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HomepageTest -->
      </class> <!-- com.test.TestNgFeatures -->
      <class name="com.test.ExceptionTimeOutTest">
        <test-method status="FAIL" signature="test1()[pri:0, instance:com.test.ExceptionTimeOutTest@55d56113]" name="test1" duration-ms="16" started-at="2018-10-11T06:44:10Z" finished-at="2018-10-11T06:44:10Z">
          <exception class="org.testng.TestException">
            <message>
              <![CDATA[
Method ExceptionTimeOutTest.test1()[pri:0, instance:com.test.ExceptionTimeOutTest@55d56113] should have thrown an exception of type class java.lang.NumberFormatException]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestException: 
Method ExceptionTimeOutTest.test1()[pri:0, instance:com.test.ExceptionTimeOutTest@55d56113] should have thrown an exception of type class java.lang.NumberFormatException
at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:80)
at org.testng.internal.Invoker.considerExceptions(Invoker.java:1139)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:612)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
      </class> <!-- com.test.ExceptionTimeOutTest -->
    </test> <!-- Different testng features test -->
  </suite> <!-- Automations on TestNg Basics -->
</testng-results>
